docs/learn/learn-teleport.md

```diff
@@ -9,30 +9,51 @@ slug: ../learn-teleport
 One of the main properties that Polkadot and Kusama bring to the ecosystems is decentralized
 blockchain interoperability. This interoperability allows for asset teleportation: the process of
 moving assets, such as coins, tokens, or NFTs, between chains (parachains) to use them as you would
-any other asset native to that chain. Interoperability is possible through [XCM][]
-and [SPREE modules][], which together ensure that assets are not lost or duplicated across multiple chain.
+any other asset native to that chain. Interoperability is possible through [XCM][] and [SPREE
+modules][], which together ensure that assets are not lost or duplicated across multiple chain.
 
 ## How to Teleport
 
+Here are a couple of things that you need to understand before submitting a teleport transaction
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically
+  deducted from the teleport amount you plan to send. Your account balance should be greater than
+  the teleport amount and the transaction fees. If you do not account for the sending fee, the
+  teleport transaction will fail.
+
+- There will be a transaction fee imposed at the destination chain. This fee is automatically
+  deducted from the teleport amount you send. After this fee is deducted, the remaining account
+  balance on the destination chain should be greater than its Existential Deposit. If not, the
+  teleport transaction will fail.
```

Comment:
can you say in both cases if the failed transaction will have tx fees charged. I am guessing there can be no charge in the first case but you will be charged both tx fees on the remote failure?

Reply:
Will test and get back to you. I will tag you once the PR is ready for review